io/private/las/Utils.cpp

```diff
@@ -515,18 +518,32 @@ void V10BaseLoader::load(PointRef& point, const char *buf, int bufsize)
 
     uint16_t intensity;
     uint8_t flags;
-    uint8_t classification;
+    uint8_t classificationWithFlags;
     int8_t scanAngleRank;
     uint8_t user;
     uint16_t pointSourceId;
 
-    istream >> intensity >> flags >> classification >> scanAngleRank >> user >> pointSourceId;
+    istream >> intensity >> flags >> classificationWithFlags >> scanAngleRank >> user >> pointSourceId;
 
     uint8_t returnNum = flags & 0x07;
     uint8_t numReturns = (flags >> 3) & 0x07;
     uint8_t scanDirFlag = (flags >> 6) & 0x01;
     uint8_t flight = (flags >> 7) & 0x01;
 
+    uint8_t classification = classificationWithFlags & 0x05;
```

Comment:
Shouldn't this be `classificationWithFlags & 0x1F;` to mask the 5 LSBs?

Reply:
I think you're correct.

Reply:
@uclaros I fixed this in #4269.  However this error had no practical effect, since it's in unused code.  Good catch though.